internal/ci/ci.go

```diff
@@ -2,8 +2,10 @@ package ci
 
 import "os"
 
-// IsAvailable detects whether this code is executed inside a CI environment.
+// IsAvailable detects whether this code is executed inside a CI environment
 func IsAvailable() bool {
 	// Most CI providers have this.
-	return os.Getenv("CI") != "" || os.Getenv("BUILD_NUMBER") != ""
+	isCi := os.Getenv("CI") != "" || os.Getenv("BUILD_NUMBER") != ""
+	shouldSkipCi := os.Getenv("SKIP_CI") != ""
```

Comment:
Looks good! Sorry, should have commented earlier, but I'd love to keep the brevity that golang encourages and simply name the var `skip` or `skipCI` here.

Feel free to merge as is though 👍